media/css/firefox/family/index.less

```diff
@@ -0,0 +1,883 @@
+// Conditional download bar
+#conditional-download-bar {
+    background: #fff;
+    color: @textColorPrimary;
+    min-height: 1.5em;
+    overflow: hidden;
+    padding: 15px 0;
+    position: relative;
+    text-align: center;
+    width: 100%;
+    .transition(all 0.25s ease);
+
+    .message {
+        display: none;
+        padding: 0 (@gridGutterWidth * 3) 0 @gridGutterWidth;
+    }
+
+    .btn-download {
+        .font-size(@smallFontSize);
+        padding: .5em 30px;
+        margin: 5px 10px 0;
+        background-color: #6fbe4a;
+
+        &:hover,
+        &:focus,
+        &:active {
+            background-color: lighten(#6fbe4a, 5%);
+        }
+
+        @media only screen and (max-width: @breakTablet) {
+            display: block;
+        }
+    }
+
+    .btn-close {
+        .image-replaced();
+        border: 0;
+        cursor: pointer;
+        height: 30px;
+        width: 30px;
+        padding: 0;
+        position: absolute;
+        right: 30px;
+        top: 10px;
+        background: #efefef url(/media/img/firefox/family/index/closex.png) center center no-repeat;
+        .at2x('/media/img/firefox/family/index/closex.png', 19px, 19px);
+
+        &:hover {
+            background-color: darken(#efefef, 5%);
+        }
+    }
+}
+
+#conditional-download-bar #dlbar-nonfx {
+    display: block;
+}
+
+html.firefox-latest {
+    #conditional-download-bar  {
+        display: none;
+    }
+}
+
+html.firefox-old {
+    #conditional-download-bar #dlbar-oldfx-desktop {
+        display: block;
+    }
+}
+
+html.ios {
+    #conditional-download-bar #dlbar-ios {
+        display: block;
+    }
+}
+
+html.android {
+    #conditional-download-bar #dlbar-nonfx-android {
+        display: block;
+    }
+}
+
+html.firefox-old,
+html.android,
+html.ios {
+    #conditional-download-bar #dlbar-nonfx {
+        display: none;
+    }
+}
+
+
+
+/*--------------------------------------------------------------------------*/
+// Mozilla tab (like tabzilla, but not)
+#moztab {
+    position: absolute;
+    right: 0;
+    top: 0;
+
+    a {
+        .image-replaced();
+        background: url(/media/img/firefox/family/index/family-sprite.png) -90px -556px no-repeat;
+        display: block;
+        width: 148px;
+        height: 36px;
+        position: relative;
+        z-index: 10;
+        .at2x('/media/img/firefox/family/index/family-sprite.png', 500px, 645px);
+    }
+
+    &:before {
+        content: '';
+        display: block;
+        width: 88px;
+        height: 26px;
+        background: @mozIDBlue;
+        position: absolute;
+        left: 28px;
+        top: 0;
+        z-index: 9;
+    }
+}
+
+
+
+/*--------------------------------------------------------------------------*/
+// Header
+#header {
+    position: relative;
+    z-index: 2;
+    height: 570px;
+    background: url(/media/img/firefox/family/index/fox.png) ~"calc(50% + 580px)" bottom no-repeat;
```

Comment:
Ugh, I noticed this page crashes IE9 when I resize the browser. Then I read this: :cry: 

    Partial support in IE9 refers to the browser crashing when used as a background-position value

Source: http://caniuse.com/#feat=calc